config/locales/en.yml

```diff
@@ -157,7 +157,7 @@ en:
         question_required: "You must answer whether the person is still serving in the UK armed forces as you told us they’re a current or former regular"
         question_not_required: "You cannot answer whether the person is still serving in the UK armed forces as you told us they’re not a current or former regular"
       preg_occ:
-        no_female: "You must answer ‘no’ as there are no females aged 16-50 in the household"
+        no_female: "You must answer ‘no’ as there are no women aged 11-65 in the household"
```

Comment:
@baarkerlounger I think we should go with 'females' as that's how we're phrasing the gender question currently - though if we end up revisiting the gender issue overall, then it might need to change